tests/test_benchmark.py

```diff
@@ -204,17 +204,19 @@ def build(self, parent, filename, args, shared_args, emcc_args, native_args, nat
       EMCC, filename,
       OPTIMIZATIONS,
       '-s', 'INITIAL_MEMORY=256MB',
-      '-s', 'FILESYSTEM=0',
       '-s', 'ENVIRONMENT=node,shell',
       '-s', 'BENCHMARK=%d' % (1 if IGNORE_COMPILATION and not has_output_parser else 0),
       '-o', final
-    ] + shared_args + emcc_args + LLVM_FEATURE_FLAGS + self.extra_args
+    ] + shared_args + LLVM_FEATURE_FLAGS
     if common.EMTEST_FORCE64:
       cmd += ['--profiling']
     else:
       cmd += ['--closure=1', '-sMINIMAL_RUNTIME']
-    if 'FORCE_FILESYSTEM' in cmd:
-      cmd = [arg if arg != 'FILESYSTEM=0' else 'FILESYSTEM=1' for arg in cmd]
+    # add additional emcc args at the end, which may override other things
+    # above, such as minimal runtime
+    cmd += emcc_args + self.extra_args
+    if 'FORCE_FILESYSTEM' not in cmd:
+      cmd += ['-s', 'FILESYSTEM=0']
```

Comment:
Why explicitly disable the filesystem here?  (i.e. why not let emcc decide it filesystem is needed or not?)

Reply:
I think the reasoning for this moved code is that most benchmarks don't use files, and power users will be disabling the filesystem entirely which can save a few more bytes in some cases.